core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/FixedRateSamplingTelemetryProcessor.java

```diff
@@ -64,20 +72,8 @@
      */
     public FixedRateSamplingTelemetryProcessor() {
         this.samplingPercentage = 100.00;
-        this.includedTypes = new HashSet<Class>();
-        this.excludedTypes = new HashSet<Class>();
-        try {
-            this.allowedTypes = new HashMap<String, Class>() {{
```

Comment:
I little bit refactored two things here, first is double brace initialization which is not considered as good practice because of creating of anonymous class. Second is using .class instead of Class#forName, there is no reason to use this as all classes are present in the core module.